### ISSUE TITLE
"Cylon" animation for installing add-ons also inflicts migraines

### ISSUE THREAD

**Issue by MarjaE2** [mozilla/addons-frontend#9351]
### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)

Go to add-ons.mozilla.org. Install a suitable add-on.

### What happened?

A dot pops up and moves back and forth. It inflicts migraines. A similar animation has complaints on the support site and the bugzilla. e.g. https://support.mozilla.org/en-US/questions/1232771

### What did you expect to happen?

Not that.

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)

---

**For QA:** animations should be disabled for both the install button and the loading indicators/texts. You need to configure the reduce setting, see: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#User_Preferences (I used the `ui.prefersReducedMotion` **number** pref set to `1`)

**Comment by ioanarusiczki** [comment]
@willdurand I added  `ui.prefersReducedMotion` in about:config with the value `1`. 

The result is no motion:

![motionless](https://user-images.githubusercontent.com/33448286/79342890-5dfc5180-7f36-11ea-9ea2-29ea83b62d36.gif)

Verified on dev with FF75(Win10).

**Comment by willdurand** [comment]
> The result is no motion:

Yes, this is expected indeed. It's hard to determine how much motion is okay so I guess it's fine. Personally, I like no animation for the loading indicators :D

**Comment by ioanarusiczki** [comment]
@willdurand Yes. Perhaps it was a bit too much before but it did not bother me.
